RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.cc

```diff
@@ -30,6 +30,7 @@ class OnlineBeamSpotESProducer : public edm::ESProducer {
 private:
   const BeamSpotOnlineObjects* compareBS(const BeamSpotOnlineObjects* bs1, const BeamSpotOnlineObjects* bs2);
   const BeamSpotOnlineObjects* checkSingleBS(const BeamSpotOnlineObjects* bs1);
+  const bool isGoodBS(const BeamSpotOnlineObjects* bs1);
```

Comment:
Shouldn't this be
```suggestion
  bool isGoodBS(const BeamSpotOnlineObjects* bs1) const;
```
(not sure there is an advantage returning a const bool in this case)

Reply:
@dzuolo can you implement this please?

Reply:
Hi @mmusich! This comment has been addressed in https://github.com/cms-sw/cmssw/pull/40744/commits/d34fe0e6031a4a6996d4ff5587c135d5c7c6fbcd

Reply:
@dzuolo you missed the `const` **after** the method :)